app/lib/apply_api/code_sets/disabilities.rb

```diff
@@ -3,7 +3,7 @@
 module ApplyApi
   module CodeSets
     module Disabilities
-      MAPPING = {
+      OLD_MAPPING = {
```

Comment:
Will we need this mapping forever?

Reply:
probably not after 2023; unless for any reason we want to import historic trainees from apply?

Reply:
It's always useful to still have the old mapping if we can, because trainees on Apply can 'defer' a year or more, and might still have the 'old style' disability data in that case when they eventually get recruited.

If we can't keep the mapping, in those cases I think we need to be making sure the mapping makes it as though disability data is missing, and require the provider to re-enter. @edwardhorsford

Reply:
Happy to write another ticket for this though if it's not currently the case.